adapters/huaweiads/huaweiadstest/exemplary/banner4_mccmnc.json

```diff
@@ -0,0 +1,269 @@
+{
+  "mockBidRequest": {
+    "id": "a3b18c5b-6708-4bb7-b41b-40ed37d89561",
+    "source": {
```

Comment:
I don't see any difference in this test and the one above. The only two differences are 
- An empty object for device.geo being specified in the above test and device.geo not being specified at all in this test, which is essentially one and the same thing.
- Difference in the MCCMNC code. One test case with a valid MCCMNC code should be enough

Reply:
@mansinahar banner4_mccmnc is to test when both `device.geo.country` and `user.geo.country` don't exist, this is to invoke the getMcc function.